shuup/core/templatetags/shuup_common.py

```diff
@@ -136,3 +137,29 @@ def json(value):
     if isinstance(value, Undefined):
         value = None
     return mark_safe(json_dump(value, cls=ExtendedJSONEncoder))
+
+
+@library.global_function
+@contextfunction
+def get_shop_configuration(context, name, default=None):
+    """
+    Get configuration variable value for the current shop.
+
+    :type context: jinja2.runtime.Context
+    :type name: str
+    :type default: Any
+    """
+    from shuup import configuration
```

Comment:
I guess not, cause tests fail with "ImportError: cannot import name 'ConfigurationItem'" if import is global